### PR TITLE
Documentation for IC2 Classic Tweaker

### PR DESCRIPTION
**Pull request by Trinsdar**
~Currently the IC2 Classic Tweaker docs are inside the IC2 Tweaker section, but I don't know whether that's where it should be.~ Decided to move it into it's own folder instead of being nested in the Ic2 tweaker folder

### CODE REVIEW

**Comment by CrankySupertoon** [comment]
I think it should be a separate catagory I made the original PR and call tell you I didn’t want to name it IC2 and EnderIO

**Comment by Trinsdar** [comment]
@kindlich sorry for getting to this so late. for the descriptions, were you just referring to the screw up I made in the canning machine doc, or was there more?

File: docs/Mods/IC2Tweaker/Scrap_Box.md [hunk 1]
```diff
@@ -7,17 +7,22 @@
...
 
-## Add Drop (IItemStack, float)
+### Add Drop (IItemStack, float)
 
 ```zenscript
 mods.ic2.ScrapBox.addDrop(IItemStack item, float weight);
 mods.ic2.ScrapBox.addDrop(<minecraft:diamond_ore>, 0.1F);
 ```
+
+### Removal
```
[the remark this answers is not in the document]
Comment (Trinsdar) [reply docs/Mods/IC2Tweaker/Scrap_Box.md:25]: Honestly I don't know why I did that

File: docs/Mods/IC2ClassicTweaker/Sawmill.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- **[IItemStack](/Vanilla/Items/IItemStack/) output**
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) input**
+
+### Addition
+
+```zenscript
+mods.ic2.Sawmill.addRecipe(IItemStack output, IIngredient input);
+
+mods.ic2.Sawmill.addRecipe(<minecraft:diamond_block>, <minecraft:dirt> * 64);
```
[the remark this answers is not in the document]
Comment (Trinsdar) [reply docs/Mods/IC2ClassicTweaker/Sawmill.md:19]: I think so, though now I need to test that

File: docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Method
+
+- **float value**
+- **[IItemStack](/Vanilla/Items/IItemStack/)... inputs** Can have more then one specified for the input. If so all the specified items give the same specified value.
+
+### Add Rare Earth Entries
+
+```zenscript
+mods.ic2.RareEarthExtractor.addRareEarthEntries(float value, IIItemStack... input);
```
[the remark this answers is not in the document]
Comment (Trinsdar) [reply docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md:17]: woops

File: docs/Mods/IC2ClassicTweaker/AdvCrafting.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+## Add Recipes
+
+### addShaped
+```zenscript
+mods.ic2.AdvRecipes.addShaped(output,inputs);
+```
+
+This creates a shaped recipe for `output` using `inputs` as Ingredients.
+
+`output` is an [IItemStack](https://docs.blamejared.com/1.12/en/#Vanilla/Items/IItemStack/)
```
[the remark this answers is not in the document]
Comment (Trinsdar) [reply docs/Mods/IC2ClassicTweaker/AdvCrafting.md:29]: I have no clue, I'll fix that

File: docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Rare Earth Extractor
+
+## Package
+
+```java
```
Comment (kindlich) [inline docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md:5]: java?

File: docs/Mods/IC2ClassicTweaker/CanningMachine.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+- **[IItemStack](/Vanilla/Items/IItemStack/) input** The item to be removed from being an input for the fuel can.
+
+
+
+See [here](https://github.com/TinyModularThings/IC2Classic/wiki/Fuel-Cans) for info on how fuel cans work.
+
+
+- **[IItemStack](/Vanilla/Items/IItemStack/) output** The product
```
Comment (kindlich) [inline docs/Mods/IC2ClassicTweaker/CanningMachine.md:77]: This belows further down?

File: docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md (hunk 5 above)
Comment (Trinsdar) [reply docs/Mods/IC2ClassicTweaker/RareEarthExtractor.md:5]: woops, must be from me doing java stuff all the time

File: docs/Mods/IC2ClassicTweaker/CanningMachine.md (hunk 6 above)
Comment (Trinsdar) [reply docs/Mods/IC2ClassicTweaker/CanningMachine.md:77]: actually turns out I forgot to move that one below the proper category

File: docs/Mods/IC2ClassicTweaker/AdvCrafting.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
+# Advanced Recipes
+
+## Package
+
+```java
```
Comment (kindlich) [inline docs/Mods/IC2ClassicTweaker/AdvCrafting.md:5]:
```suggestion
```zenscript
```

**Pull request merged**